src/sass/components/_header.scss

```diff
@@ -173,29 +173,86 @@ header {
         display: flex;
         align-items: center;
         position: relative;
+        cursor: pointer;
+        &:hover{
+          background: $primary;
+          &:hover ul li ul {
+            display: block;
+            opacity: 1;
+            visibility: visible;
+          }
+          &:hover ul li, &:hover i {
+            color: $white;
+          }
+        }
 
-        i:first-child {
+        i {
           color: $primary;
           position: absolute;
           left: 10px;
           z-index: 0;
         }
 
-        i:last-child {
-          position: absolute;
-          right: 10px;
-          z-index: 0;
-        }
-
-        select {
-          -webkit-appearance: none;
-          border: 0;
-          background-color: transparent;
-          outline: none;
-          padding: 5px 30px 5px 35px;
-          font-size: 14px;
+        ul {
+          text-align: left;
+          display: inline;
+          margin: 0;
+          padding: 0px 30px;
+          list-style: none;
           position: relative;
           z-index: 1;
+          li{
```

Comment:
You should format only newly added code to make sure it's consistent. For now - let it be as is, but please keep it in mind in the future.